src/experiments.js

```diff
@@ -1,3 +1,17 @@
+/**
+ * Utils.
```

Comment:
We should have more descriptive docblocks here. Not only does this not explain what the function does, it's potentially misleading.

Reply:
Oh yeah, that docblock was for going to be for a section of the file rather than the function. I'll update now